src/index.html

```diff
@@ -11,7 +12,142 @@
+      method="post"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+        <label>Surname:
+          <input 
+            type="text" 
+            name="surname" 
+            required 
+            autocomplete="off"
+          >
+        </label>
+
+        <label>Name:
+          <input 
+            type="text" 
+            name="name" 
+            required 
+            autocomplete="off"
+          >
+        </label>
+
+        <label>How old are You?
+          <input 
+            type="number" 
+            name="age" 
+            maxlength="3" 
+            minlength="2" 
+            value="12" 
+            min="1" max="100"
+          >
+        </label>
+
+        <label>Full date of birth:
+          <input 
+            type="date" 
+            name="birthdate" 
+            placeholder="mm/dd/yyyy"
+          >
+        </label>
+
+        <label>I accept the term of the agreement
+          <input type="checkbox" name="terms">
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>Registration</legend>
+        <label>E-mail:
+          <input 
+            type="email" 
+            name="email" 
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label>Password:
+          <input type="password" name="pass">
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+        <label class="inline radioWrap">Do you love cats?
+          <input 
+            class="radio" 
+            type="radio" 
+            name="lovecats" 
+            value="love"
+          >Yes
+        </label>
+        <label class="inline radioWrap">
+          <input 
+            class="radio" 
+            type="radio" 
+            name="lovecats" 
+            value="doNotLove"
+          >No
+        </label>
+
+        <label>What is your favorite color?
+          <input 
+            type="color" 
+            name="choosecolor" 
+            value="black"
+          >
+        </label>
+
+        <label>What time do you go to bed?
+          <input 
+            type="time" 
+            name="bedtime" 
+            placeholder="--:--:--"
+          >
+        </label>
+
+        <label>What are your favorite brand of cars?
+          <select name="carbrand" multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+
+        <label>How do you rate our work?
+          <input 
+            type="range" 
+            name="rate" 
+            min="1" max="10"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>Additional info:</legend>
+        <label>Comments:
+          <textarea 
+            name="comment" 
+            cols="20" rows="2"
+          >
+          </textarea>
```

Comment:
p8 in [Checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)

```suggestion
         ></textarea>
```